### ISSUE TITLE
Meteor M2-3

### ISSUE THREAD

**Issue by creinemann** [dernasherbrezon/r2cloud#140]
Will Meteor M 2-3 be added with decoding?

**Comment by dernasherbrezon** [comment]
Yes, I think the current version is based on old meteor modulation qpsk and won’t produce results. Will add support in the nearest future

**Comment by dernasherbrezon** [comment]
Fixed in 2bbd952. Meteor should be available through satdump integration.
